helm_deploy/values-dev.yaml

```diff
@@ -12,6 +12,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     SENTRY_ENVIRONMENT: dev
+    SERVICES_PRISONS-API_BASE-URL: https://prison-api-dev.prison.service.justice.gov.uk
```

Comment:
Can we just call this `PRISON_API_URL` to match the structure of `HMPPS_AUTH_URL` above? We'll need to do this across the three envs.